src/StartMenu.ts

```diff
@@ -296,64 +298,85 @@ export class AEntryPointList implements IEntryPointList {
   private updateList(data: INamedSet[]) {
     const that = this;
 
+    const headerText = this.desc.name === 'Genes'? 'Gene Sets' : `${this.desc.name.substr(0, this.desc.name.length - 1)} Panels`;
```

Comment:
ordino itself doesn't know that the presented data are 'genes' or 'panels'. It is in this case maybe but no in the general one